test/gerkin/tests/features/data_read_value.feature

```diff
@@ -0,0 +1,16 @@
+Feature: API endpoint allowing users to search for one record's field value in the Digital Registries database.
+  Request endpoint: GET /data/{registryname}/{versionnumber}/{uuid}/read-value/{field}.{ext}
+
+  Background:
+    Given The user wants to search for the first name of the user with uuid "123" in the Digital Registries database
+
+
+  Scenario: The user receives the first name of searches user from the Digital Registries database
+    When The user triggers an action to receive the first name of searches user from the database
+    And The request with a valid payload is sent
```

Comment:
Valid payload isn't the only condition here, the entry needs to exist in the database for this scenario and for the other one the searched entry doesn't exist in the database.